pyproject.toml

```diff
@@ -53,7 +53,6 @@ yamllint = "*"
 toml = "*"
 # Python implementation of markdownlint
 pymarkdownlnt = [
-    {version = "0.9.26", python = "~3.8.0"},
     {version = "~0.9.29", python = ">=3.9,<3.13"},
 ]
```

Comment:
Probably can just remove the version qualifier here? `pymarkdownlnt = "~0.9.29"`?